code/build.gradle

```diff
@@ -2,7 +2,7 @@
 
 buildscript {
     ext {
-        kotlin_version = '1.4.0'
+        kotlin_version = '1.7.20'
```

Comment:
Note: These versions will be updated to be used from Gradle Plugin in subsequent PR for 3.x migration